website/www/site/layouts/_default/baseof.html

```diff
@@ -13,7 +13,7 @@
 <!DOCTYPE html>
 <html lang="{{ .Site.Language.Lang }}" class="no-js">
 <head>
-  {{ partial "head.html" . }}
+  {{ partial "head_homepage.html" . }}
```

Comment:
I believe this will alter the default template for every page type in the site. Is that what we want? Could you use an override template for the homepage, and make this change there?

Reply:
Yeah, thank you for the good point, after consultation on hugo forum came with another approach. 
https://discourse.gohugo.io/t/add-different-head-html-to-homepage/39133
The commiter to hugo suggest using `if else` statement to choose between head and head_homepage, but i can also create `baseof.html` in layouts/baseof.html folder. Which one would you reccomend?

Reply:
It's your call. I don't think we want to have a lot of if/else logic in the templates. That gets unwieldy. But if it's just one statement, you could do it either way.

Reply:
Came with the solution of separate baseof.html in `layouts/baseof.html`, so that would affect only homepage.